examples/ansible/roles/alp_reboot/templates/myplaybook.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=My Ansible Playbook
+After=network.target
+
+[Service]
+Type=oneshot
+WorkingDirectory={{ playbook_path }}
+ExecStart=ansible-playbook {{ ansible_playbook_basename }}  -e 'playbook_path={{ playbook_path }}' -e 'ansible_user={{ ansible_user }}' -e 'reboot_mode=auto'
```

Comment:
Note that the enablement of auto reboot when calling the ansible playbook in this service could lead to "infinite" reboots if the user have a logic bug in the resumed playbook that keeps configuring additional reboot resumptions...

You may want to make the reboot_mode setting that is passed to the resumed playbook something that the user of the role can specify/override if they want to avoid such a problem scenario.